README.md

```diff
@@ -283,6 +214,4 @@ I would like to thank...
 
```

Comment:
- [ ] please like the previous reviewer noted, your MIT link is broken. kindly update it. we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

Reply:
Thanks, I used MIT licence for the template.